metabrainz/utils.py

```diff
@@ -1,6 +1,8 @@
 from urllib.parse import urlparse, parse_qsl, urlunparse, urlencode
-import string
+
 import random
+import string
```

Comment:
Is this needed?

Reply:
I moved the imports around to make them alphabetical. The import is  used in a the [generate_string](https://github.com/metabrainz/metabrainz.org/blob/81fc69a531080928f41657f6a755f4827ddd4820/metabrainz/utils.py#L14) function.